tensorflow/training/docker/2.9/py3/cu112/Dockerfile.gpu

```diff
@@ -0,0 +1,375 @@
+FROM nvidia/cuda:11.2.2-base-ubuntu20.04 AS common
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# TensorFlow major.minor version 
+ENV TF_VERSION=2.9
+
+# prevent stopping by user interaction
+ENV DEBIAN_FRONTEND noninteractive
+ENV DEBCONF_NONINTERACTIVE_SEEN true
+
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONIOENCODING=UTF-8
+ENV LANG=C.UTF-8
+ENV LC_ALL=C.UTF-8
+
+# Set environment variables for MKL
+# For more about MKL with TensorFlow see:
+# https://www.tensorflow.org/performance/performance_guide#tensorflow_with_intel%C2%AE_mkl_dnn
+ENV KMP_AFFINITY=granularity=fine,compact,1,0
+ENV KMP_BLOCKTIME=1
+ENV KMP_SETTINGS=0
+ENV RDMAV_FORK_SAFE=1
+
+ARG PYTHON=python3.9
+ARG PIP=pip3
+ARG PYTHON_VERSION=3.9.10
+
+ARG OPEN_MPI_PATH=/opt/amazon/openmpi
+ARG EFA_PATH=/opt/amazon/efa
+ARG EFA_VERSION=1.12.1
+ARG OMPI_VERSION=4.1.1
+ARG BRANCH_OFI=1.2.0-aws
+
+ARG CUDA=11.2
+ARG CUDA_DASH=11-2
+ARG CUDNN=8.1.0.77-1
```

Comment:
The CUDNN version is also not updated? where did you get this version from? CUDNN from TF2.8 isCUDNN=8.2.4.15-1